src/App.tsx

```diff
@@ -15,71 +15,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [chosenElement, chosenGood] = useState(goods[8]);
```

Comment:
Pay attention to the task description. Keep the proper naming.
Also, setState function should be named according to the noun variable
![image](https://user-images.githubusercontent.com/96409563/232120187-75ab26ac-b27b-47dc-a4f0-e3e3bd2523cf.png)